models/record.go

```diff
@@ -455,6 +459,31 @@ func (rc *RecordConfig) ToRR() dns.RR {
 	return rr
 }
 
+func (rc *RecordConfig) GetNameFQDN() string {
```

Comment:
The dnssort package uses an interface named `SortableChange` which has `GetNameFQDN`, `GetType` and `GetFQDNDependencies`. Using the interface instead of a direct dependency on RecordConfig makes it more reusable for other uses if need be (and a bit easier to test)